doc/manual/src/design/store/entries.md

```diff
@@ -0,0 +1,46 @@
+# Store Entries
+
+File system data in Nix is organized into *store entries*.
+A store entry is the combination of
+
+  - some file system data
+  - references to store entries
+
+## File system data
+
+Nix supports the a similar model of the file system as Git.
```

Comment:
I found this sentence pretty confusing. Maybe something like, "The nix store uses a simple filesystem model, similar to Git: ..."

Reply:
```suggestion
The nix store uses a simple filesystem model, similar to the one Git uses.
```